packages/editor-ui/src/components/Banner.vue

```diff
@@ -19,7 +19,14 @@
 					>
 						{{ message }}&nbsp;
 					</span>
-					<n8n-link v-if="details && !expanded" :bold="true" @click="expand">More details</n8n-link>
+					<n8n-link
+						v-if="details && !expanded"
+						:bold="true"
+						size="small"
+						@click="expand"
+					>
+						<span :class="$style.moreDetails">More details</span>
```

Comment:
maybe link component should be expanded to support xsmall size instead

Reply:
There's a bit of an inconsistency here: `n8n-link` passes the size to `n8n-text`, but `n8n-text` size resolves weird:
- `medium` is `font-size-s`
- `small` is `font-size-2xs`
- `xsmall` is `font-size-3xs`

Essentially, the `font-size-xs` that I need is skipped. 😰